zerver/lib/bugdown/api_arguments_table_generator.py

```diff
@@ -117,11 +117,18 @@ def render_table(self, arguments: List[Dict[str, Any]]) -> List[str]:
 
             # TODO: OpenAPI allows indicating where the argument goes
             # (path, querystring, form data...).  We should document this detail.
+            example = ""
+            if 'example' in argument:
+                example = escape_html(json.dumps(argument['example']))
```

Comment:
I believe `json.dumps` is always wrong in this case. The default serialization rules are very different from JSON.

Reply:
Changed.